common/morphisms/pipeline.ts

```diff
@@ -210,21 +247,21 @@ export const getReportDataObj = (
   });
 };
 
-const buildStageData: schema.PipelineStepData = {
-  temperature: 0,
-  batchSize: 0,
-  tokenCount: {
-    sent: 0,
-    received: 0,
-    total: 0,
-  },
-  costPerToken: {
-    denomination: "$",
-    value: 0,
-  },
-  model: "claude-instant-v1",
-  instructions: "",
-};
+// const buildStageData: schema.PipelineStepData = {
```

Comment:
@Brandon-Perry can you add a comment about why this is left in but commented out, if keeping it isn't an error